swank/match.lisp

```diff
@@ -158,17 +158,17 @@
         ((consp pattern)
          (append
           `((consp ,key))
-          (compile-select-tests (!cs-car key) (car
```

Comment:
I am. But this file wasn't part of my initial pull request. I don't understand how it got added...
This should be local to my fork. Not that I wouldn't mind have the change incorporated...